CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+*   The `render_component` test helper has been renamed to `render_inline`.
+
+    *Joel Hawksley*
+
+*   Components are now rendered with `render MyComponent, foo: :bar` syntax. The existing `render MyComponent.new(foo: :bar)` syntax will be deprecated in version 2.0.
+
+    *Joel Hawksley*
+
 *   Components now inherit from ActionView::Component::base
```

Comment:
I think there's a typo here: `base` -> `Base`

Also, I find it helpful when changelogs contain a heading with the version number that separates each list of changes that were made in that version - would you be open to adding that?

Reply:
Resolved by 8c95681867cd9fef19fc626bb215a1de705a3534 and 06310bb4761a0520494c312834631de695a2774d